<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yeastar-PC" name="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" tests="9" failures="3" timestamp="26 Oct 2018 03:30:25 GMT" time="302.687" errors="0">
  <testcase name="A1_add_callback" time="73.869" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
  <testcase name="A0_Register" time="11.159" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
  <testcase name="B_editInbound1" time="30.304" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
  <testcase name="G_Recovery" time="28.746" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
  <testcase name="C_calldefault2_sps" time="27.668" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback">
    <failure type="java.lang.AssertionError" message="预期2000会响铃 【预期：2 实际：4】">
      <![CDATA[java.lang.AssertionError: 预期2000会响铃 【预期：2 实际：4】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.driver.SwebDriver.ysAssertWithHangup(SwebDriver.java:912)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback.C_calldefault2_sps(Callback.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- C_calldefault2_sps -->
  <testcase name="C_calldefault1_sip" time="27.709" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback">
    <failure type="java.lang.AssertionError" message="预期3001会响铃 【预期：2 实际：0】">
      <![CDATA[java.lang.AssertionError: 预期3001会响铃 【预期：2 实际：0】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.driver.SwebDriver.ysAssertWithHangup(SwebDriver.java:912)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback.C_calldefault1_sip(Callback.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- C_calldefault1_sip -->
  <testcase name="F_delete" time="46.450" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
  <testcase name="E_callbackThroughSPS" time="27.709" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback">
    <failure type="java.lang.AssertionError" message="预期辅助2的2000分机响铃 【预期：2 实际：4】">
      <![CDATA[java.lang.AssertionError: 预期辅助2的2000分机响铃 【预期：2 实际：4】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.driver.SwebDriver.ysAssertWithHangup(SwebDriver.java:912)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback.E_callbackThroughSPS(Callback.java:138)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E_callbackThroughSPS -->
  <testcase name="D_editInbound1" time="29.073" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback -->
